recommender/engines/autoencoders/training/data_validation_step.py

```diff
@@ -34,7 +34,7 @@ class AEDataValidationStep(DataValidationStep):
 
     def __init__(self, pipeline_config):
         super().__init__(pipeline_config)
-        self.least_num_of_usr_srv = self.resolve_constant(LEAST_NUM_OF_USR_SRV, 1)
+        self.least_num_of_usr_srv = self.resolve_constant(LEAST_NUM_OF_USR_SRV, 2)
```

Comment:
Maybe we should be more strict with the default values? 100 for example?

Reply:
Generally, we can - the more the better. On the other hand, then it will be difficult to satisfy this condition locally (through swagger queries), although this is not a priority for sure.